src/components/KRadio/KRadio.vue

```diff
@@ -63,18 +64,23 @@
         <slot />
       </span>
       <span
-        v-if="label"
-        class="radio-label"
+        v-if="label || showCardDescription"
+        class="card-label-container"
       >
-        {{ label }}
-      </span>
-      <span
-        v-if="showCardDescription"
-        class="radio-description"
-      >
-        <slot name="description">
-          {{ description }}
-        </slot>
+        <span
+          v-if="label"
+          class="radio-label"
+        >
+          {{ label }}
+        </span>
+        <span
+          v-if="showCardDescription"
+          class="radio-description"
+        >
+          <slot name="description">
+            {{ description }}
+          </slot>
+        </span>
```

Comment:
For this whole section (unsure if I highlighted all the correct lines) should we utilize `div` instead of `span`? 

Since the user can slot any content into the `description` slot, we need to support block-level elements

Reply:
I don't think we should because here's what we say in the docs:

![Screenshot 2024-11-07 at 3 47 55 PM](https://github.com/user-attachments/assets/b1121bd2-3011-46f2-8665-24f59db4613e)